internal/configs/module.go

```diff
@@ -64,6 +64,9 @@ type Module struct {
 	// IsOverridden indicates if the module is being overridden. It's used in
 	// testing framework to not call the underlying module.
 	IsOverridden bool
+
+	// TODO: is it a good idea to put StaticEvaluator inside a Module?
+	StaticEvaluator *StaticEvaluator
```

Comment:
This is what I was leaning towards, though it might not be the most elegant solution.

The only other solution I can come up with is to construct a new static evaluator from the root config's module in meta_encryption.go

Reply:
I think `Module` is a better fit for `StaticEvaluator` since these values are more coherent.